src/test/mir-opt/const_allocation3/i686/rustc.main.ConstProp.after.mir

```diff
@@ -32,8 +32,8 @@ alloc2 (static: FOO, size: 4, align: 4) {
 
 alloc9 (size: 142, align: 1) {
     0x00 │ ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab
-    0x10 │ ab ab ab ab ab ab ab ab ab ab ab ab ╾alloc6+0─╼
-    0x20 │ 01 ef cd ab 00 00 00 00 00 00 00 00 00 00 00 00
+    0x10 │ ab ab ab ab ab ab ab ab ab ab ab ab ab ab ╾alloc6+0 (4 ptr bytes)
+    0x20 │ ────╼ 01 ef cd ab 00 00 00 00 00 00 00 00 00 00
```

Comment:
I didn't mean the lines, actually. I meant the overflow *within* a line, which is much easier to showcase on 32bit than 64bit.

Reply:
ah oops, done

Reply:
Thanks! Looks about as horrible as I imagined, but I don't have any better suggestions. ;)